src/GoogleLogin.js

```diff
@@ -25,7 +25,7 @@ export default class GoogleLogin extends React.Component {
         if (!gapi.auth2.getAuthInstance()) {
```

Comment:
Instead of having `true` be hardcoded could we have it set to `this.propsfetchBasicProfile`? 
cc @Maggicorp this is what I meant by passing it as a prop, but I didn't explain well

Reply:
-__- it didn't show the piece of code I wanted to be previewed, it's [this](https://github.com/easylo/react-google-login-component/blob/patch-1/src/GoogleLogin.js#L25-L31)